www/obs/index.html

```diff
@@ -12,9 +12,7 @@
         Gurmukhi: 'ik®pw kry jy AwpxI qw hir rKw aur Dwir ]1]',
         English: 'If the Lord shows His Mercy, then I keep Him enshrined within my heart. ||1||',
         Punjabi: 'jy prmwqmw AwpxI myhr kry, qW mYN BI aus dw nwm ihrdy ivc pro r`KW [1[',
-        Transliteration: {
-          English: 'kirapaa kare je aapanee taa har rakhaa aur dhaar ||1||',
-        },
+        Transliteration: 'kirapaa kare je aapanee taa har rakhaa aur dhaar ||1||',
```

Comment:
is this how actual lines are sent? does this allow for different translits in the future?

Reply:
I will check andn have it mimic the actual object.